src/V3Options.h

```diff
@@ -452,7 +449,10 @@ class V3Options final {
     bool debugSelfTest() const { return m_debugSelfTest; }
     bool decoration() const { return m_decoration; }
     bool dpiHdrOnly() const { return m_dpiHdrOnly; }
-    bool dumpDefines() const { return m_dumpDefines; }
+    bool dumpDefines() const {
+        const auto it = m_dumpLevel.find("defines");
+        return it != m_dumpLevel.end() && it->second > 0;
+    }
```

Comment:
Note unlike the other dumps this is used by users.  I don't think this changes anything, but I'm not sure you noticed.

Reply:
Thanks, didn't realise that, though it should just work.